x-pack/plugins/viz_editor/public/common/components/field_list_panel/field_list_panel.tsx

```diff
@@ -48,12 +48,12 @@ export function FieldListPanel({
   visModel,
   onChangeVisModel,
   getSuggestionsForField,
-}: PanelComponentProps) {
+}: VisualizationPanelProps) {
   const datasource = visModel.datasource;
   const [state, setState] = useState(() => initialState());
 
   function filterFields(field: DatasourceField) {
-    return field.name.includes(state.fieldsFilter);
+    return field.name.toLowerCase().includes(state.fieldsFilter.toLowerCase());
```

Comment:
This fixes the filtering bug, it's not actually related to the other work here...